Sources/Common/DataModel/Cylinder/index.js

```diff
@@ -0,0 +1,103 @@
+import macro from 'vtk.js/Sources/macro';
+import vtkMath from 'vtk.js/Sources/Common/Core/Math';
+
```

Comment:
I would import our Constants like that `import Constants from 'vtk.js/Sources/Common/DataModel/Cylinder/Constants';`

Reply:
Cylinder does not need any constants, why import them?

Reply:
So they can be exposed in the vtkCylinder namespace.

But I most likely commented on the wrong file... ;-)